lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -262,7 +263,7 @@ def _package_dict(package):
         'release': package.release,
         'version': package.version,
         'repo': package.repoid}
-    result['nevra'] = '{epoch}:{name}-{version}-{release}.{arch}'.format(
+    result['envra'] = '{epoch}:{name}-{version}-{release}.{arch}'.format(
```

Comment:
This is abbreviated nevra in common usage so we shouldn't change to envra unless there's a compelling reason.